geometry_msgs/Point[] local_points
